src/jag-vips-client/src/main/java/ca/bc/gov/open/jagvipsclient/payment/PaymentServiceImpl.java

```diff
@@ -16,29 +19,45 @@
  */
 public class PaymentServiceImpl implements PaymentService {
 
-	private final PaymentStatusApi paymentApi;
+	private final PaymentStatusApi paymentStatusApi;
```

Comment:
Question: Can these two APIs be merged somehow?

Reply:
I think we will be able to use both the apis in the PaymentService class that we have in digital forms.

Reply:
Kind of what I was thinking.. OK, thanks, I'll merge this.